isstools/gui.py

```diff
@@ -501,6 +502,22 @@ def save_trajectory(self):
             else:
                 print('\n.txt is not a valid name')
 
+    def plot_traj_file(self):
+        self.traj.load_trajectory_file('/GPFS/xf08id/trajectory/' + self.comboBox.currentText())
```

Comment:
Tom, I think I can (need to write the code)... I think Eli wanted to keep parsing the files like we are doing for some reason, but I can't remember why.